### PR TITLE
DM-44386: Test every page endpoint for loading success

### PR DESCRIPTION
**Pull request by ugyballoons**
Also test for invalid dates

### CODE REVIEW

File: tests/handlers/external_test.py [hunk 1]
```diff
@@ -91,3 +96,89 @@ async def test_current_channels(
...
+    client, app, mocker = mocked_client
+
+    locations = [loc.name for loc in m.locations]
+    summit: Location = find_first(m.locations, "name", "summit-usdf")
+    assert isinstance(summit, Location)
+
+    all_cams = [f"{summit.name}/{cam.name}" for cam in summit.cameras]
+    online_cams = [f"{summit.name}/{cam.name}" for cam in summit.cameras if cam.online]
+    camera_historical = [f"{cam}/historical" for cam in online_cams]
+    camera_dates = [f"{cam}/date/{day_obs}" for cam in online_cams if cam]
```
Comment (sebastian-aranda) [inline tests/handlers/external_test.py:115]: Why does only this case use the `if cam` checking? (when iterating over `online_cams`).

File: tests/handlers/external_test.py [hunk 2]
```diff
@@ -91,3 +96,89 @@ async def test_current_channels(
...
+    ]
+
+    for url_frag in invalid_urls:
+        url = f"/{app_name}/{url_frag}"
+        res = await client.get(url)
+        assert res.is_error
+
+    # TODO: This needs not to be just a success, but a page with a
+    # 'nothing for this day' message.
+    # See DM-45327 https://rubinobs.atlassian.net/browse/DM-45327
```
Comment (sebastian-aranda) [inline tests/handlers/external_test.py:180]: We closed this ticket, so should this part change? at least remove the TODO perhaps? If there is still work to do, please add another ticket :pray:.

**Review by sebastian-aranda — APPROVED** [review]
Looks good to me :+1:. I've left a couple comments, but the PR is approved :wink:.

File: tests/handlers/external_test.py (hunk 1 above)
Comment (ugyballoons) [reply tests/handlers/external_test.py:115]: Good spot, thanks! It's redundant.

**Pull request merged**